Setup Hello Microverse Project :blush:

HAMMAS-SALEEM opened the pull request:
### This is my first pull request for my first repository in Microverse Program. This is my Hello Microverse Project and i have done following things in it. :memo:

:heavy_check_mark: Added a markup file (HTML)
:heavy_check_mark: Added a stylesheet file (CSS)
:heavy_check_mark: Added a .gitignore file
:heavy_check_mark: Added test.md file in .gitignore file
:heavy_check_mark: Setup HTML, CSS linters
:heavy_check_mark: Added a descriptive Readme file
:heavy_check_mark: Added license for this project

HAMMAS-SALEEM committed:
Add .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Add test.md for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Update README.md & index.html for Hello-Microverse

HAMMAS-SALEEM committed:
Add .stylelintrc.json for refactor-authentication

HAMMAS-SALEEM committed:
Add .hintrc for Hello-Microverse

HAMMAS-SALEEM committed:
Add linter for Hello-Microverse

HAMMAS-SALEEM committed:
Delete test.md from Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Add test.md for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-MIcroverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-MIcroverse

HAMMAS-SALEEM committed:
Delete .gitignore from Hello-Microverse

HAMMAS-SALEEM committed:
Add .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Delete test.md from Hello-Microverse

HAMMAS-SALEEM committed:
Update .gitignore for Hello-Microverse

HAMMAS-SALEEM committed:
Fix linter in style.css from Hello-Microverse

gabrielyea commented on README.md:
```diff
@@ -45,15 +45,10 @@ To get a local copy up and running follow these simple example steps.
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/HAMMAS-SALEEM)
+- Twitter: [@twitterhandle](https://twitter.com/HammasSaleem4)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/hammas-saleem-407)
```
- [x] Please remember to change the handles to match your own! That way your README looks more professional!

HAMMAS-SALEEM replied:
Sir,
     First of all, I would like to give appreciations to you to give a review of my code. But I want to say that I have my .gitignore file in my repo in my feature branch but I did not merge them yet and the .gitignore file is indicating which files should be ignored. Also, in some previous commits, I deleted the file test.md but then I added it in my project in screenshots below you can see. Pardon, I forgot to commit when I added it again but it is in my project you can see in screenshots below. You can match gitignore file in my repo in the feature branch with my commits which are after the test.md delete. You will see this file there and the code there. Now, i'm working on other changes so if you see any commit so you can verify i am working on other things.

![01](https://user-images.githubusercontent.com/93039359/143066580-d0a7a653-a0ad-427e-bc48-445f9449bc32.PNG)
![02](https://user-images.githubusercontent.com/93039359/143066601-bcfbfe99-1f17-4dfe-b2e4-179bb3c45cf7.PNG)
![03](https://user-images.githubusercontent.com/93039359/143066605-9cc3d63a-3ee9-4e03-9dd8-811ce9a4113c.PNG)
:

gabrielyea replied:
Yes, don't worry about it! I just saw your commit history but did not check your file so the .gitignore is good to go! :+1:

gabrielyea reviewed (changes requested):
# Required changes :recycle: 
Hi @HAMMAS-SALEEM, 

This is a great start! 
First, let me point out:
- Great commit history! :+1: 
- You are following gitflow. :clap: 

However, some small details are missing in your project take a look :arrow_down: 

## Required Changes ♻️
There are some issues that you still need to work on to go to the next project but you are almost there!
- [x] The `.gitignore` file should be in the repo! I saw you tried to include it but forgot to specify the files it should ignore.
Please make sure your `.gitignore` looks like this

![image](https://user-images.githubusercontent.com/28903404/143042746-d1898a1b-f6be-41a7-89ca-200ce0c70d02.png)
The files you see here will be ignored once you push them to GitHub, there is no need to manually delete them.

---

- [x] You should not delete the `test.md` It will be automatically ignored thanks to the `.gitignore`
 
---

- [x]  One of the requirements is to present the README in a professional way! I can see you are using the microverse template but you forgot to fill in your own info! Please give your README a title, put your contact info, and a short description of this project :star: 

## This is a little example:
![image](https://user-images.githubusercontent.com/28903404/143046217-fbc2fa0d-af09-4815-a706-0f5e79d6aecb.png)

---

- [x] Your `MIT license` file is not complete! Please add your name and the app type. (It can be 'web application')

---

- [x] You forgot to add a `description` to your PR, please add one.
If you are not sure how to do it here is an example:
![image](https://user-images.githubusercontent.com/28903404/143049254-d71d429d-94bf-4e58-9067-8aecaffad848.png)

---

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] 

- [x] Your PR title is not meaningful enough. Maybe something like `Setting up project tools` or `Created first feature branch` would be better? but that's up to you!

--- 

# Once you implement all the changes you should be ready to go :rocket: :rocket: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAMMAS-SALEEM committed:
Update MIT for Hello-Microverse

HAMMAS-SALEEM commented:
Sir,
First of all, I would like to give appreciations to you to give a review of
my code. But I want to say that I have my .gitignore file in my repo in my
feature branch but I did not merge them yet and the .gitignore file is
indicating which files should be ignored. Also, in some previous commits, I
deleted the file test.md but then I added it in my project in screenshots
below you can see. Pardon, I forgot to commit when I added it again but it
is in my project you can see in screenshots below. You can match gitignore
file in my repo in the feature branch with my commits which are after the
test.md delete. You will see this file there and the code there. Now, i'm
working on other changes so if you see any commit so you can verify i am
working on other things.
[image: 01.PNG][image: 02.PNG][image: 03.PNG]

On Tue, Nov 23, 2021 at 8:23 PM Gabriel Rendon ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Required changes ♻️
>
> Hi @HAMMAS-SALEEM <https://github.com/HAMMAS-SALEEM>,
>
> This is a great start!
> First, let me point out:
>
>    - Great commit history! 👍
>    - You are following gitflow. 👏
>
> However, some small details are missing in your project take a look ⬇️
> Required Changes ♻️
>
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
>    - The .gitignore file should be in the repo! I saw you tried to
>    include it but forgot to specify the files it should ignore.
>    Please make sure your .gitignore looks like this
>
> [image: image]
> <https://user-images.githubusercontent.com/28903404/143042746-d1898a1b-f6be-41a7-89ca-200ce0c70d02.png>
> The files you see here will be ignored once you push them to GitHub, there
> is no need to manually delete them.
> ------------------------------
>
>    - You should not delete the test.md It will be automatically ignored
>    thanks to the .gitignore
>
> ------------------------------
>
>    - One of the requirements is to present the README in a professional
>    way! I can see you are using the microverse template but you forgot to fill
>    in your own info! Please give your README a title, put your contact info,
>    and a short description of this project ⭐
>
> This is a little example:
>
> [image: image]
> <https://user-images.githubusercontent.com/28903404/143046217-fbc2fa0d-af09-4815-a706-0f5e79d6aecb.png>
> ------------------------------
>
>    - Your MIT license file is not complete! Please add your name and the
>    app type. (It can be 'web application')
>
> ------------------------------
>
>    - You forgot to add a description to your PR, please add one.
>    If you are not sure how to do it here is an example:
>    [image: image]
>    <https://user-images.githubusercontent.com/28903404/143049254-d71d429d-94bf-4e58-9067-8aecaffad848.png>
>
> ------------------------------
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> [OPTIONAL]
>
>    - Your PR title is not meaningful enough. Maybe something like Setting
>    up project tools or Created first feature branch would be better? but
>    that's up to you!
>
> ------------------------------
> Once you implement all the changes you should be ready to go 🚀 🚀
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/HAMMAS-SALEEM/Hello-Microverse/pull/1#discussion_r755236938>
> :
>
> > +- GitHub: ***@***.***(https://github.com/HAMMAS-SALEEM)
>
> +- Twitter: ***@***.***(https://twitter.com/HammasSaleem4)
>
> +- LinkedIn: [LinkedIn](www.linkedin.com/in/hammas-saleem-407)
>
>
>
>    - Please remember to change the handles to match your own! That way
>    your README looks more professional!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HAMMAS-SALEEM/Hello-Microverse/pull/1#pullrequestreview-813815378>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWF2V7765EOLYUDYZANQAH3UNOWYVANCNFSM5ITSJJCQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

HAMMAS-SALEEM committed:
Update MIT for Hello-Microverse(After 1st Submission)

HAMMAS-SALEEM committed:
Update README.md for Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md for Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md for Hello-Microverse(After first Submission)

HAMMAS-SALEEM commented:
Thanks for reaching out to me. One more thing the README.md file is in my
feature branch with my information now I am completing my information in it.

On Tue, Nov 23, 2021 at 9:50 PM Gabriel Rendon ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/HAMMAS-SALEEM/Hello-Microverse/pull/1#discussion_r755328037>
> :
>
> > +- GitHub: ***@***.***(https://github.com/HAMMAS-SALEEM)
> +- Twitter: ***@***.***(https://twitter.com/HammasSaleem4)
> +- LinkedIn: [LinkedIn](www.linkedin.com/in/hammas-saleem-407)
>
> Yes, don't worry about it! I just saw your commit history but did not
> check your file so the .gitignore is good to go!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HAMMAS-SALEEM/Hello-Microverse/pull/1#discussion_r755328037>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWF2V7ZISNSJCNQU3O4KM4DUNPA6HANCNFSM5ITSJJCQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update MIT.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update MIT.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update MIT.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

HAMMAS-SALEEM committed:
Update README.md fro Hello-Microverse(After first Submission)

andersonlebon commented on README.md:
```diff
@@ -1,59 +1,31 @@
-> One paragraph statement about the project.
+> My First Official Microverse Repository 
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
```
##[OPTIONAL]

- [ ] It is good that you have followed the microverse template while building your README file. However, you forgot to change the screenshot. if you have one kindly update your readme otherwise delete it from your REPO.

andersonlebon replied:
## [OPTIONAL]
- [ ] It is good that you have followed the microverse template while building your README file. However, you forgot to change the screenshot. if you have one kindly update your readme otherwise delete it from your REPO.

andersonlebon commented on index.html:
```diff
@@ -4,6 +4,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
     <title>Document</title>
```
# [OPTIONAL]
- [ ] The content on the `<title></title>` tag should describe the content of the HTML page. for example `<title>hello microverse</title>`

andersonlebon commented on README.md:
```diff
@@ -1,59 +1,31 @@
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
## [OPTIONAL]
- [ ] Kindly provide a valid ` livedemo` link or just delete it as the link is broken

andersonlebon reviewed (approved):
Hi @HAMMAS-SALEEM 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 
![nodding-yes](https://user-images.githubusercontent.com/65068771/143095390-344d1cb0-b853-4b91-8662-5d9049dc9e60.gif)


## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
